index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
```

Comment:
The tags on lines 4-7 are all tab-indented, where the rest of the document is space-indented.  You should changes these indents to spaces to match the rest of the document.